.github/workflows/lint.yml

```diff
@@ -0,0 +1,48 @@
+name: Linting workflow
+on:
+  push:
+    branches:
```

Comment:
I don't think so we need to specify `branches` when you want to run the workflow on all the branches anyway. I would still suggest to test the workflow on your fork and remove this line and 7th after that.

Reply:
Thanks for the suggestion. Updated and it is running as expected still.